pennylane_rigetti/qc.py

```diff
@@ -45,15 +46,17 @@ class QuantumComputerDevice(RigettiDevice, ABC):
         device (str): the name of the device to initialise.
         shots (int): number of circuit evaluations/random samples used
             to estimate expectation values of observables.
-         wires (Iterable[Number, str]): Iterable that contains unique labels for the
+        wires (Iterable[Number, str]): Iterable that contains unique labels for the
             qubits as numbers or strings (i.e., ``['q1', ..., 'qN']``).
             The number of labels must match the number of qubits accessible on the backend.
-            If not provided, qubits are addressed as consecutive integers ``[0, 1, ...]``, and their number
-            is inferred from the backend.
+            If not provided, qubits are addressed by the backend.
```

Comment:
To make sure I understand, does this mean that the user can pass _any_ wires and they'll be mapped to the device wires, or that the user must use the appropriate wires for the device?